nautobot/core/tests/test_api.py

```diff
@@ -630,10 +642,11 @@ def test_related_by_attributes(self):
 
     def test_related_by_attributes_no_match(self):
         data = {
-            "vid": 100,
-            "name": "Test VLAN 100",
+            "vid": 120,
+            "name": "Test VLAN 120",
```

Comment:
This change appears to make the check at line 644/657 below invalid.

These are a weird set of tests in any case though, seems like very basic data validation testing...?